### ISSUE TITLE
Timeout during import of monitors

### ISSUE THREAD

**Issue by joberp** [DataDog/datadog-sync-cli#126]
**Overview of issue:**
Timeout during import of monitors

```
datadog-sync import --config config.txt -v --resources monitors --max-workers 1
2023-04-25 17:20:39,064 - INFO - Importing monitors
2023-04-25 17:20:39,066 - DEBUG - Starting new HTTPS connection (1): xxxxx.datadoghq.com:443
2023-04-25 17:21:09,461 - ERROR - Error while importing resources monitors: HTTPSConnectionPool(host='xxxxx.datadoghq.com', port=443): Read timed out. (read timeout=30)
2023-04-25 17:21:09,462 - INFO - Finished importing monitors: 0 successes, 0 errors
```

**Steps to reproduce the issue:**
1. See command above 

**Describe the results you received:**
* None, due to timeout

**Describe the results you expected:**
* Downloaded monitors

**Additional information you deem important (e.g. issue happens only occasionally):**
* Problem occurs since 17.04.2023
* Download of dashboards works fine

**Comment by therve** [comment]
You shouldn't use your custom domain for the sync, can you try with `api.datadoghq.com`? Thanks.

**Comment by joberp** [comment]
Tanks for your quick reply .
Tried it again with with `api.datadoghq.com` :
```
datadog-sync import --config config-copy.txt -v --resources monitors --max-workers 1
2023-04-26 11:25:17,376 - INFO - Importing monitors
2023-04-26 11:25:17,377 - DEBUG - Starting new HTTPS connection (1): api.datadoghq.com:443
2023-04-26 11:25:47,802 - ERROR - Error while importing resources monitors: HTTPSConnectionPool(host='api.datadoghq.com', port=443): Read timed out. (read timeout=30)
2023-04-26 11:25:47,802 - INFO - Finished importing monitors: 0 successes, 0 errors
```

Same problem. Again, dashboards work fine.

**Comment by therve** [comment]
Can you contact support? We'd need more information about your environment that we should't share here. Thank you.

**Comment by skarimo** [comment]
Hello, we just released `0.3.0` which allows users to modify the request timeout. Could you increase the timeout using `--http-client-timeout` and let us know if this resolves the timeout issue. Thanks

**Comment by joberp** [comment]
Setting the new flag to 60 seconds solves my problem.
Downloading 50 monitors takes around 50 seconds in my case.

Thanks a lot!
